testfiles/AddHeatSourceTerm_RefBldgSmallOfficeNew2004.idf

```diff
@@ -0,0 +1,4899 @@
+! RefBldgSmallOfficeNew2004_HeatSourceTerm.idf
+!
```

Comment:
This does not match the file name.  I know the comment block below is copied from RefBldgSmallOfficeNew2004_Chicago but it's not the expected format. If you look at ExampleFiles.html, the last column "Description" is blank for all of the RefBldg files.  Please add a `Highlights` section describing the main reason for this example file.  See 5ZoneAirCooledConvCoef.idf.

@Myoldmopar It would be nice to move the Description to column 2 in ExampleFiles.html.

Reply:
The name was modified and we also added the highlight section.